packages/browser-tests/src/server.ts.new

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable no-console */
```

Comment:
why this file has extension `ts.new`? :)

Reply:
removed, unnecessary artifact